scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -20,15 +20,28 @@ class SensorVersionsFailure(Exception):
 
 gkecluster=GKECluster("compat-test")
 
+if len(chart_versions) > 1:
+    os.environ["CENTRAL_CHART_VERSION"] = chart_versions[0]
+
 failing_sensor_versions = []
 for version in chart_versions:
     os.environ["SENSOR_CHART_VERSION"] = version
     try:
         make_compatibility_test_runner(cluster=gkecluster).run()
     except Exception:
-        print(f"Exception \"{Exception}\" raised in compatibility test for sensor version {version}")
+        print(f"Exception \"{Exception}\" raised in compatibility test for sensor version {version} and central version {chart_versions[0]}")
         failing_sensor_versions += version
 
+if len(chart_versions) > 1:
+    os.environ["SENSOR_CHART_VERSION"] = chart_versions[0]
+    os.environ["CENTRAL_CHART_VERSION"] = chart_versions[1]
+    try:
+        make_compatibility_test_runner(cluster=gkecluster).run()
```

Comment:
It seems to me that try/make_compatibility_test_runner/except code here and above is nearly the same.
It would be nice to get rid of this code duplication.

You can achieve this by constructing a list of tuples/structs where each represents a pair of central version + sensor version to be tested. Then, you can iterate this list once and so have only one `make_compatibility_test_runner` call (in the loop).